homeassistant/components/prometheus/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Prometheus",
   "documentation": "https://www.home-assistant.io/integrations/prometheus",
   "requirements": ["prometheus_client==0.7.1"],
-  "dependencies": ["http"],
+  "dependencies": ["http", "humidifier"],
```

Comment:
I am not sure why this is needed (but not other integrations such as `light` or `climate`...

Reply:
```
diff --git a/script/hassfest/dependencies.py b/script/hassfest/dependencies.py
index 2335270dc4..d9e1b7b923 100644
--- a/script/hassfest/dependencies.py
+++ b/script/hassfest/dependencies.py
@@ -122,6 +122,7 @@ ALLOWED_USED_COMPONENTS = {
     "cover",
     "device_tracker",
     "fan",
+    "humdifier",
     "image_processing",
     "light",
     "lock",
```

This should allow it ^^

Reply:
Thanks! Trying that...

Reply:
That works! Thanks)